scripts/webpack.config.js

```diff
@@ -73,6 +73,7 @@ function getConfig(isReactExternalized) {
                 examples: path.join(__dirname, '../examples/src'), // for examples only
                 'box-ui-elements-locale-data': path.resolve(`i18n/${language}`),
                 'box-locale-data': path.resolve(`node_modules/@box/cldr-data/locale-data/${language}`),
+                lodash : path.resolve('node_modules/lodash'),
```

Comment:
had to put this in because it was looking for lodash/merge within blueprint when it wasnt there

Reply:
I'm not clear on why this is needed since lodash is already present in package.json. Do we have a type mismatch? Or a misconfigured dependency in Blueprint?